go.mod

```diff
@@ -32,7 +32,7 @@ require (
 	k8s.io/api v0.29.4
 	k8s.io/apimachinery v0.29.4
 	k8s.io/client-go v0.29.4
-	k8s.io/utils v0.0.0-20241210054802-24370beab758
+  k8s.io/utils v0.0.0-20241210054802-24370beab758
```

Comment:
I guess there was an accidental intervention here. Could you please run `go mod tidy`?